http/courseEntryHandler.go

```diff
@@ -43,9 +43,17 @@ func (a *AppServer) PostCourseEntryHandler() httprouter.Handle {
 			return
 		}
 
-		pURLs, err := url.URLifyStrings(request.Pictures)
+		err, paths := a.CourseEntryService.StoreCourseEntryFiles(request.Pictures, id, request.Date)
```

Comment:
This seems unusual to me. Aren't files usually uploaded using `multipart/form-data`? Also, what kind of data is the file content here? Is is Base64 encoded? I did not see any decoding in this PR. As a last thought, there is no limit on file size.

Reply:
the json decoder decodes the base64 encoded bytes-file automagically. 
The filesize is not limited yet.